config.inc.php

```diff
@@ -28,7 +29,16 @@
     'PMA_UPLOADDIR',
     'PMA_SAVEDIR',
     'PMA_SSL',
+    'PMA_SSL_VERIFY',
+    'PMA_SSL_CA',
+    'PMA_SSL_KEY',
+    'PMA_SSL_CERT',
     'PMA_SSLS',
+    'PMA_SSL_VERIFIES',
+    'PMA_SSL_CAS',
+    'PMA_SSL_KEYS',
+    'PMA_SSL_CERTS',
+    'PMA_PMA_SSL_DIR'
```

Comment:
```suggestion
    'PMA_SSL_DIR'
```
this should be `PMA_SSL_DIR`

Reply:
My mistake